Lib/fontTools/ttLib/__init__.py

```diff
@@ -299,8 +300,11 @@ def saveXML(self, fileOrPath, progress=None, quiet=None,
 			if progress:
 				progress.set(i)
 			tag = tables[i]
-			if splitTables:
+			if splitTables or (splitGlyphs and (tag == 'glyf')):
```

Comment:
the parentheses around `(tag == 'glyf')` are redundant

Reply:
...but it makes order of operation explicit which can be nice for readability.

Reply:
well, it's obvious (to me at least) the precedence of `==` operator over `and`.